model/sharing/revisions_test.go

```diff
@@ -347,17 +347,6 @@ func TestIndexerCreateBogusPrevRev(t *testing.T) {
 	assert.Equal(t, indexer.bulkRevs.Rev, rev)
 }
 
-func TestConflictName(t *testing.T) {
-	assert.Equal(t, "foo (2)", conflictName("foo", false))
-	assert.Equal(t, "foo (3)", conflictName("foo (2)", false))
-	assert.Equal(t, "foo (1000)", conflictName("foo (999)", false))
-	assert.Equal(t, "foo () (2)", conflictName("foo ()", false))
-
-	assert.Equal(t, "bar (2)", conflictName("bar", true))
-	assert.Equal(t, "bar (2).txt", conflictName("bar.txt", true))
-	assert.Equal(t, "bar (3).txt", conflictName("bar (2).txt", true))
-}
-
```

Comment:
It is not possible to mock an indexer to keep the test?

Reply:
Well, technically, it is possible. But it's a lot of work for not a lot of value.

Reply:
Ok, I think it's a bit of a shame to lose it, as it also helps to understand the behavior. Maybe at least a comment in the function header, would help, like: 
```
A conflicted file `foo` will be renamed foo (2), then foo (3), etc
```